airgun/browser.py

```diff
@@ -236,11 +237,20 @@ class AirgunBrowserPlugin(DefaultPlugin):
         }
         '''
 
+    success_on_timeout = False
+
     def ensure_page_safe(self, timeout='30s'):
         """Ensures page is fully loaded.
         Default timeout was 10s, this changes it to 30s.
+        If self.success_on_timeout is True, the function doesn't raise an exception
+        and continues as if the page was safe instead. This can be used to bypass
+        some bugs, e.g. https://bugzilla.redhat.com/show_bug.cgi?id=2106022
         """
-        super().ensure_page_safe(timeout)
+        try:
+            super().ensure_page_safe()
+        except TimedOutError as e:
+            if not self.success_on_timeout:
+                raise e
```

Comment:
No need to capture the exception object here, you can drop `as e` and `raise e`, just calling `raise` will propagate the exception object.